crypto/dsa/dsa_asn1.c

```diff
@@ -32,6 +32,15 @@ void DSA_SIG_get0(BIGNUM **pr, BIGNUM **ps, const DSA_SIG *sig)
     *ps = sig->s;
 }
 
+int DSA_SIG_set0(BIGNUM *r, BIGNUM *s, DSA_SIG *sig)
```

Comment:
Oh... it matches the getters :( Are the getters new in 1.1.0? If so, we should fix them!